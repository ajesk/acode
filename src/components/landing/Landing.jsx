import React from 'react';
import './Landing.scss';

const Landing = () => {
  return (
    <div className="landing">
      <h1>
        Simple === better
      </h1>
      <p>
        Nothing is better than simple easy to understand code/architecture
      </p>
    </div>
  );
}

export default Landing;
