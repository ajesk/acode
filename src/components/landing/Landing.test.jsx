import React from 'react';
import { render } from '@testing-library/react';
import Landing from './Landing';

describe('landing', () => {
  test('snapshot', () => {
    expect(render(<Landing />)).toMatchSnapshot();
  });
});