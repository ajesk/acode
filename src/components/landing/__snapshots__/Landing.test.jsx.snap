// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="landing"
      >
        <h1>
          Simple === better
        </h1>
        <p>
          Nothing is better than simple easy to understand code/architecture
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="landing"
    >
      <h1>
        Simple === better
      </h1>
      <p>
        Nothing is better than simple easy to understand code/architecture
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
